test/test_helper.rb

```diff
@@ -0,0 +1,7 @@
+if ENV['CI'] == 'true'
+  require 'simplecov'
+  SimpleCov.start
+
+  require 'codecov'

```

Comment:
In order to be able to require the codecov library you should install this gem before as well.